charts/karavi-observability/Chart.yaml

```diff
@@ -4,3 +4,8 @@ name: karavi-observability
 description: Karavi Observability is part of the [Karavi](https://github.com/dell/karavi) open source suite of Kubernetes storage enablers for Dell EMC storage products. Karavi Observability provides Kubernetes administrators with visibility into metrics and topology data related to containerized storage.
 type: application
 version: 0.2.0
+dependencies:
+- name: cert-manager
+  version: 1.1.0
+  repository: https://charts.jetstack.io
+  condition: certManager.enabled
```

Comment:
Should this condition be removed so that we always install cert-manager? If enabled was set to false, wouldn't that prevent the observability helm chart from being installed properly due to the dependency on cert-manager?

Reply:
You are right. Updated